odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,23 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
-
-Properties with a defined default value, nullable properties, and
+Non-insertable properties SHOULD be omitted from the request body.
+If they are provided, services MUST either ignore the value in the request body or fail the request.
+
+Non-insertable properties include (and are not limited to)
+- dependent properties that are tied to
+non-key properties of the principal entity through a referential constraint (informally: "denormalized" properties),
```

Comment:
```suggestion
- dependent properties with a referential constraint whose principal property lies outside the entity key (informally: "denormalized" properties),
```

Reply:
Then keep it here and append the following sentence to the first paragraph in [OData-CSDL, section 8.5]:
> We say that the dependent property and the principal property are _tied to_ each other.

And link the words "referential constraint" to that section.

The term "tied to" also appears in [OData-CSDL, section 8.6].